src/renderer/services/wallet/balances.ts

```diff
@@ -51,7 +54,24 @@ export const reloadBalances$: Rx.Observable<O.Option<LoadBalancesHandler>> = sel
 /**
  * Transforms BNB data (address + `AssetsWB`) into `AssetsWBChain`
  */
-const bnbAssetsWBChain$: Observable<ChainBalance> = Rx.combineLatest([BNB.address$, BNB.balances$, network$]).pipe(
+const thorBalance$: Observable<ChainBalance> = Rx.combineLatest([THOR.address$, THOR.balances$]).pipe(
```

Comment:
just curious. dont we have to be dependent on selected network ?

Reply:
Underlaying `Client` used by `THOR.address$` and `THOR.balances$` is already depending on selected network. Any change by `network$`[ will be handled in `clientState$`](https://github.com/thorchain/asgardex-electron/blob/65f393ed30d658d99c6470bc21eb4f84ed0a269b/src/renderer/services/thorchain/common.ts#L35).